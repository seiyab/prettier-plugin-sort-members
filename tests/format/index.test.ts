import { describe, test, expect } from "bun:test";
import { readdir, readFile } from "node:fs/promises";
import { join } from "node:path";
import { format } from "prettier";

describe("format", async () => {
	const dir = join(import.meta.dir, "testdata");
	const filenames = await readdir(dir);
	test.each(filenames)("%s", async (name) => {
		const path = join(dir, name);
		const code = await readFile(path, "utf-8");
		const result = await format(code, { filepath: path });
		expect(result).toMatchSnapshot();
	});
});
