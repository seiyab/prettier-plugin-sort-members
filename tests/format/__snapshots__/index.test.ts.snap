// Bun Snapshot v1, https://goo.gl/fbAQLP

exports[`format interface.ts 1`] = `
"interface A {
  b: string;
  c: unknown;
  a: string;
}
"
`;
